### ISSUE TITLE
emotion 6.0.2 throws error when interpolating values into `css`

### ISSUE THREAD

**Issue by rgdelato** [tkh44/emotion#164]
- `emotion` version: 6.0.2
- `react` version: 15.4.2

Issue is fixed by reverting to `emotion` version 6.0.1.

Relevant code.

```javascript
import {css} from "emotion";

const inputClass = css`
  position: ${'relative'};
`;
```

What happened:

`Uncaught TypeError: forEach is not a function`

<img width="555" alt="screen shot 2017-07-18 at 3 32 51 pm" src="https://user-images.githubusercontent.com/805850/28342793-d01ac734-6bce-11e7-9346-fa163a96d6ec.png">
<img width="555" alt="screen shot 2017-07-18 at 3 32 28 pm" src="https://user-images.githubusercontent.com/805850/28342795-d472893e-6bce-11e7-9836-fc3dfca1dd6d.png">

Suggested solution:

It looks like the import for `forEach` was changed just recently, so I'm worried that I'm missing context around the issue, but it seems like the issue is something involving that?

**Comment by tkh44** [comment]
This should have been fixed in `6.0.2`. Let me check on it.

**Comment by tkh44** [comment]
Fixed in `6.0.3`

**Comment by rgdelato** [comment]
Awesome, this appears to be fixed, closing this issue.
